public/API/API_GetGameExtended.php

```diff
@@ -50,28 +50,97 @@
  *    string    Expiration                date the claim will expire
  */
 
+use App\Community\Models\AchievementSetClaim;
 use App\Platform\Enums\AchievementFlag;
+use App\Platform\Models\Achievement;
+use App\Platform\Models\Game;
+use Carbon\Carbon;
 
-$gameID = (int) request()->query('i');
+$gameId = (int) request()->query('i');
 $flag = (int) request()->query('f', (string) AchievementFlag::OfficialCore);
-getGameMetadata($gameID, null, $achData, $gameData, flag: $flag, metrics: true);
 
-if ($gameData === null) {
+if ($game = Game::with('system')->find($gameId)) {
```

Comment:
Assignment operators within conditionals can be cumbersome to understand. Please assign `$game` outside of an `if`, then perform the truthiness check within.

Reply:
All further responses now rely on $game returning data